Src/ScipBe.Common.Office.OneNote/ScipBe.Common.Office.OneNote.nuspec

```diff
@@ -2,16 +2,19 @@
 <package>
   <metadata>
     <id>ScipBe.Common.Office.OneNote</id>
-    <version>2.0.2</version>
+    <version>2.0.3</version>
```

Comment:
I would prefer to change version to 3.0. Changing the properties to static is a breaking change, so use semver to make that clear

Reply:
Oh yes I definitely agree! This change was actually part of the other PR (adding package dependencies), but I can update the ver here too instead of in a separate PR, I'll do that...